packages/monocdk-experiment/.npmignore

```diff
@@ -0,0 +1,24 @@
+# The basics
+*.ts
+*.tgz
+*.snk
+!*.d.ts
+!*.js
+
+# Coverage
+coverage
+.nyc_output
+.nycrc
+
+# Build gear
+dist
+.LAST_BUILD
+.LAST_PACKAGE
+.jsii
+
+
+# Include .jsii
+!.jsii
```

Comment:
![Please don't do this to me](https://media.tenor.com/images/c8763ba06feb4e5ee6e8f3649bd7bb8c/tenor.gif)

Reply:
Fixed